controllers/comment-controllers.js

```diff
@@ -0,0 +1,17 @@
+const { deleteCommentById } = require("../models/comment-models");
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteCommentById(comment_id)
+    .then((msg) => {
+      res.status(204).send({ msg });
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
Nice, but like your test suggests we cannot send any data back so sending msg here isnt actually doing anything. I would recommend using `res.sendStatus(204)` as an alternative here as it doesnt need the .send(...) method at the end of it and looks cleaner.